resources/views/partials/flash-messages.blade.php

```diff
@@ -2,7 +2,7 @@
     <x-live-region>
         @if (flash()->message)
             <x-hearth-alert :type="flash()->class" :title="Str::contains(flash()->message, __('You are previewing your')) ? __('Draft') : null">
-                {!! Str::markdown(flash()->message) !!}
+                {{ safe_markdown(flash()->message) }}
```

Comment:
As discussed, we could change this line to:

    {{ flash()->message }}

And then always pass text rather than Markdown.

We'd need to change 2 files where we currently use markup to render a link in the flash message:

- `EngagementController.php`
- `HasMultipageEditingAndPublishing.php`